sql/migrations/warehouse/000021_alter_wh_schemas_wh_upload_id_column.up.sql

```diff
@@ -0,0 +1,5 @@
+--
+-- wh_schemas
+--
+
+ALTER TABLE wh_schemas ALTER COLUMN wh_upload_id drop default;
```

Comment:
Got this bug while writing test cases, we are not setting wh_upload_id when inserting to wh_schemas. Currently, it is being populated as part of [AUTO INCREMENT](https://github.com/rudderlabs/rudder-server/blob/9a80f459f5289df3d99d25fc7b09ed1804bd6522/sql/migrations/warehouse/000001_create_tables.up.sql#L130).